doc/pyoptsparse.bib

```diff
@@ -0,0 +1,558 @@
+    year      = {2020},
+    doi       = {10.2514/6.2020-1967},
+    keywords  = {OpenAeroStruct}
+}
+
+@inproceedings{Kruger2020,
+    title     = {The Feasibility of Electric Propulsion for Commuter Aircraft},
+    author    = {Kruger, Michael and Uranga, Alejandra},
+    booktitle = {AIAA Scitech 2020 Forum},
+    pages     = {1499},
+    year      = {2020},
+    doi       = {10.2514/6.2020-1499},
+    keywords  = {SUAVE}
+}
+
+@inproceedings{karpuk2021conceptual,
+    title     = {Conceptual Design Trade Study for an Energy-Efficient Mid-Range Aircraft with Novel Technologies},
+    author    = {Karpuk, Stanislav and Elham, Ali},
+    booktitle = {AIAA Scitech 2021 Forum},
+    pages     = {0013},
+    year      = {2021},
+    doi       = {10.2514/6.2021-0013},
+    keywords  = {SUAVE}
+}
+
+@article{Zahle2016,
+    author   = {Frederik Zahle and Carlo Tibaldi and Christian Pavese and Michael K. McWilliam and Jose P. A. A. Blasques and Morten H. Hansen},
+    journal  = {Journal of Physics: Conference Series},
+    keywords = {OpenMDAO},
+    number   = {6},
+    pages    = {062008},
+    title    = {Design of an Aeroelastically Tailored 10 {MW} Wind Turbine Rotor},
+    volume   = {753},
+    year     = {2016}
+}
+
+@inproceedings{Thomas2018,
+    address      = {Milano, Italy},
+    author       = {Thomas, Jared J. and Ning, Andrew},
+    booktitle    = {Journal of Physics: Conference Series},
+    doi          = {10.1088/1742-6596/1037/4/042012},
+    keywords     = {OpenMDAO},
+    month        = {June},
+    number       = {042012},
+    organization = {The Science of Making Torque from Wind},
+    title        = {A Method for Reducing Multi-Modality in the Wind Farm Layout Optimization Problem},
+    volume       = {1037},
+    year         = {2018}
+}
+
+@inproceedings{Ivaldi2015a,
+    address   = {Dallas, TX},
+    author    = {Davide Ivaldi and Ney R. Secco and Song Chen and John T. Hwang and Joaquim R. R. A. Martins},
+    booktitle = {Proceedings of the 16th AIAA/ISSMO Multidisciplinary Analysis and Optimization Conference},
+    doi       = {10.2514/6.2015-3436},
+    month     = {June},
+    title     = {Aerodynamic Shape Optimization of a Truss-Braced-Wing Aircraft},
+    year      = {2015},
+    keywords  = {MACH}
+}
+
+@inproceedings{Rajnarayan2018a,
+    author    = {Dev Rajnarayan and Andrew Ning and Judd A. Mehr},
+    booktitle = {Proceedings of the AIAA Aviation Forum},
+    doi       = {10.2514/6.2018-3949},
+    title     = {Universal Airfoil Parametrization Using B-Splines},
+    year      = {2018}
+}
+
+@article{Madsen2019a,
+    author  = {Mads H. Aa. Madsen and Frederik Zahle and Niels N. S{\o}rensen and Joaquim R. R. A. Martins},
+    doi     = {10.5194/wes-4-163-2019},
+    journal = {Wind Energy Science},
+    month   = {April},
+    pages   = {163--192},
+    title   = {Multipoint high-fidelity {CFD}-based aerodynamic shape optimization of a 10 {MW} wind turbine},
+    volume  = {4},
+    year    = {2019}
+}
```

Comment:
I suppose this paper kinda used MACH-Aero, although for a novel application

Reply:
Right forgot about that, thanks